app/models/user.rb

```diff
@@ -34,6 +34,8 @@ class User < ActiveRecord::Base
   validates :email, uniqueness: true, allow_blank: true,
                     format: {with: /\A[^@\s]+@[^@\s]+\Z/}
 
+  validates :email, fake_email: true, :if => lambda { Rails.env.staging? }
```

Comment:
only on staging ??

Reply:
Yup.. Only for testing .. Right now.

we were thinking to not block fake emails for staging and we will block only for prod.

Any thoughts? Is it okay to block for both environments? 🤔

Reply:
I think we should do it for both. As we have tested emails and we are getting them properly.

Reply:
Okay.. Great. adding it for both environments.